x-pack/filebeat/input/netflow/netflow_test.go

```diff
@@ -148,7 +154,7 @@ func readDatTests(t testing.TB) *DatTests {
 	return &tests
 }
 
-func getFlowsFromDat(t testing.TB, name string, datFiles ...string) TestResult {
+func getFlowsFromDat(t testing.TB, name string, testData DatTest) TestResult {
```

Comment:
Nit: This mixed naming in var/funcs named `dat` vs `data` is confusing. Both are right, but I think if you pick one or the other across the board, things will be more clear. I think "data" better describes it.